templates/.github/.OwlBot.lock.yaml

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 docker:
   image: gcr.io/cloud-devrel-public-resources/owlbot-java:latest
-  digest: sha256:1ec28a46062b19135b11178ceee60231e5f5a92dab454e23ae0aab72cd875906
-  # created: 2022-06-27T15:01:06.405564326Z
+  digest: sha256:b47e411068ffc6e6990cb4e0802fbac9a4488dec7f5f424553518ba21abea208
```

Comment:
I got the latest sha256 from gcr.io/cloud-devrel-public-resources/owlbot-java:latest